Userland/Libraries/LibGUI/Event.h

```diff
@@ -509,4 +509,24 @@ class ActionEvent final : public Event {
     NonnullRefPtr<Action> m_action;
 };
 
+inline const char* mouse_button_to_string(MouseButton key)
+{
+    switch (key) {
+    case MouseButton::None:
+        return "None";
+    case MouseButton::Primary:
+        return "Primary";
+    case MouseButton::Secondary:
+        return "Secondary";
+    case MouseButton::Middle:
+        return "Middle";
+    case MouseButton::Backward:
+        return "Backward";
+    case MouseButton::Forward:
+        return "Forward";
+    default:
+        return nullptr;
```

Comment:
Should we just use `VERIFY_NOT_REACHED()` here?

Reply:
Mm I originally had that but figured I should probably copy how `KeyCode`'s handled unknown cases, but can change (and I guess update the caller to not check for `nullptr` 👍